### ISSUE TITLE
[OSX] The characters entered are periodically duplicated when text is entered in any available input field

### ISSUE THREAD

**Issue by DmitrySProductengine** [secondlife/viewer#2901]
### Environment
<details><summary>Environment</summary>
<p>

Second Life Release 7.1.12.11386494836 (64bit)
Release Notes

You are at 125.7, 243.7, 50.6 in Product Engine 3 located at simhost-0baa562fa277f8a2d.agni
SLURL: http://maps.secondlife.com/secondlife/Product%20Engine%203/126/244/51
(global coordinates 336766.0, 307188.0, 50.6)
Second Life Server 2024-09-13.10853867644
Release Notes

CPU: Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (2600 MHz)
Memory: 16384 MB
OS Version: macOS 15.0.1 Darwin 24.0.0 Darwin Kernel Version 24.0.0: Tue Sep 24 23:36:30 PDT 2024; root:xnu-11215.1.12~1/RELEASE_X86_64 x86_64
Graphics Card Vendor: ATI Technologies Inc.
Graphics Card: AMD Radeon Pro 555X OpenGL Engine

OpenGL Version: 4.1 ATI-6.1.13

Window size: 1920x1200
Font Size Adjustment: 96pt
UI Scaling: 1
Draw distance: 128m
Bandwidth: 3000kbit/s
LOD factor: 1.25
Render quality: 2
Texture memory: 4096MB
Disk cache: Max size 1638.4 MB (17.1% used)
HiDPI display mode: 

RestrainedLove API: (disabled)
J2C Decoder Version: KDU v7.10.4
Audio Driver Version: OpenAL, version 1.1 ALSOFT 1.23.1 / OpenAL Community / OpenAL Soft: OpenAL Soft
Dullahan: 1.14.0.202408091638
  CEF: 118.4.1+g3dd6078+chromium-118.0.5993.54
  Chromium: 118.0.5993.54
LibVLC Version: 3.0.21
Voice Server Version: Vivox 4.10.0000.32327.5fc3fe7c.5942f08

Packets Lost: 1/2086 (0.0%)
October 21 2024 02:32:07

</p>
</details> 

### Reproduction steps

1. Download and install the Second Life Release 7.1.12.11386494836 (64bit) viewer (https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%230ef7a9b-maint-c-2024-10-17).
2. Log in to the Viewer.
3. Open any input field (for example - Nearby chat).
4. Try entering some text.

**Actual result:**
The characters entered are periodically duplicated when text is entered in any available input field.

**Expected result:**
The characters entered should not duplicated when text is entered in any available input field.

**Additional information:**
This issue was found during Ad hoc testing in the scope of [IQA #368](https://github.com/secondlife/iqa/issues/368).
This issue is reproduced on OSX.
This issue is not reproduced on Win10 and Win11.

This issue is not reproduced on the current release 7.1.10.10800445603 build.

See the attached video (the video shows duplicate symbols, although they are entered only once):
1. Actual - OSX - 7.1.12.11386494836:

https://github.com/user-attachments/assets/beff2f38-aa7b-4e7c-8248-e7a74f379715

**Comment by Dan-Linden** [comment]
Duplicate of https://github.com/secondlife/viewer/issues/2870
